quickwit-core/src/index.rs

```diff
@@ -114,6 +116,28 @@ pub async fn delete_index(
     Ok(deleted_entries)
 }
 
+/// Cleans up split cache in local split store.
+///
+/// * `metastore_uri` - The metastore URI for accessing the metastore.
+/// * `index_id` - The target index Id.
+/// * `grace_period` -  Threshold period after which a staged split can be garbage collected.
+/// * `dry_run` - Should this only return a list of affected files without performing deletion.
+pub async fn clean_split_cache(
+    data_dir_path: &Path,
+    index_id: String,
+    source_id: String,
+) -> anyhow::Result<()> {
+    let cache_path = data_dir_path
+        .join("indexing")
+        .join(source_id)
+        .join(index_id)
+        .join("cache");
```

Comment:
"indexing" and "cache" are easy to break on changes, can you move them to variables and use them everywhere?